src/databricks/labs/remorph/reconcile/constants.py

```diff
@@ -60,11 +63,36 @@ class ReportType(AutoName):
 class ThresholdMode(AutoName):
     PERCENTILE = "percentile"
     ABSOLUTE = "absolute"
-
-
-class ThresholdMatchType(AutoName):
-    INTEGER = "integer"
-    TIMESTAMP = "timestamp"
+    NUMBER_ABSOLUTE = "number_absolute"
+    NUMBER_PERCENTILE = "number_percentile"
+    DATETIME = "datetime"
+
+
+class ThresholdSQLTemplate(AutoName):
+    SELECT_NUMBER_ABSOLUTE = """source.{column} as {column}_source, databricks.{column} 
+                                as {column}_databricks, case when (coalesce(source.{column},0) - coalesce(databricks.{column},0)) == 
```

Comment:
```suggestion
                                AS {column}_databricks, CASE WHEN (COALESCE(source.{column}, 0) - COALESCE(databricks.{column},0)) == 
```

let's use uppercase for SQL keywords to maintain readability.

Reply:
My bad this is all stale code after sqlglot expressions are used. Removed the code.